### DISCUSSION TITLE
关于Knit qmd到html的error

### DISCUSSION THREAD

**Discussion by yang7er** [rbind/yihui#1524] [Q&A] [answered]
Hi，
最近这两次在knit图片，动图，表格去到html上老报这个错，但是我不知道是我那里出了问题，代码是对的，是我需要下什么东西吗？
<img width="1280" alt="Screenshot 2023-10-07 at 7 53 28 pm" src="https://github.com/yihui/yihui.org/assets/101859927/b951d1c1-acc9-43aa-b3a4-961b39798d18">

**Comment by yang7er** [comment]
因为看到他说我的error在xfun的grep_sub里。十分期待回答，感谢yihui老师 T...T。一个周四想要用精美动图slides做pre的学生留～

**Comment by yihui** [comment, marked as answer]
八字箴言：[遇到问题，先试升级](https://yihui.org/en/2017/05/when-in-doubt-upgrade/)。

https://d.cosx.org/d/424744

**Reply by yang7er** [reply]
解决啦解决啦，感谢yihui老师！！这个已经困扰我一礼拜了！记住八字箴言了！！！
